docs/core/integration-quality-scale/rules/iqs021.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "IQS021 - Clean up stale devices"
+---
+
+IQS021 is as follows:
+
+Clean up stale devices
+
+## Reasoning
+
+When a device is removed from a hub or account, it should also be removed from Home Assistant.
+This way, the user interface will not show devices that are no longer available.
+
+We should only remove devices that we are sure are no longer available.
```

Comment:
I think integrations that can't say this for sure should implement the `async_remove_config_entry_device` so users can delete them themselves as a last resort. Should this be in the rule or should this be in the exceptions

Reply:
I'd put it in the rule.